jwtauthenticator/jwtauthenticator.py

```diff
@@ -12,30 +12,51 @@ def get(self):
         header_name = self.authenticator.header_name
         auth_header_content = self.request.headers.get(header_name, "")
         signing_certificate = self.authenticator.signing_certificate
+        secret = self.authenticator.secret
         username_claim_field = self.authenticator.username_claim_field
         audience = self.authenticator.expected_audience
+        tokenParam = self.get_argument('token', default=False)
```

Comment:
According to [this RFC](https://tools.ietf.org/html/rfc6750#section-2.3) the query parameter should be 'access_token'. Any further thoughts on this? I'm guessing you've added it this way due to some application config specific to your use case? 

If it's configurable then i'd rather adhere to the RFC on this. If you've got valid objections though then i'm happy to merge.